setup.cfg

```
@@ -32,12 +32,12 @@ packages = find:
 python_requires = >=3.7
 install_requires =
     torch>=1.12
-    bitsandbytes==0.34.0
+    bitsandbytes==0.37.1
     accelerate==0.15.0
     huggingface-hub==0.11.1
     transformers==4.25.1
```

```suggestion
    transformers>=4.25.1,<5.0.0
```

This is requested in Discord (since people would like to use newer transformer features), approved by @mryab (he says that the transformers' backward compatibility is good enough).